source/features/align-issue-labels.tsx

```diff
@@ -12,5 +57,6 @@ https://github.com/pulls
 https://github.com/bmish/eslint-doc-generator/pulls
 https://github.com/bmish/eslint-doc-generator/issues
 https://github.com/bmish/eslint-doc-generator/milestone/1
```

Comment:
I'm just replacing the URL.

```suggestion
https://github.com/bmish/eslint-doc-generator/milestone/1?closed=1
```

I assume the feature is broken there, I wouldn't worry about it in this PR.

Reply:
it does work 

![image](https://github.com/user-attachments/assets/ecd1e278-d3e7-45c0-83c8-5ba0e6a2e064)